backend/process.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from transformers import pipeline
+
+summarizer = pipeline(
+    "summarization",
+    "pszemraj/long-t5-tglobal-base-16384-book-summary",
+    device=0 if torch.cuda.is_available() else -1,
```

Comment:
Conventionally, it is preferred to use "cuda" and "cpu" strings for devices
```python
"cuda" if torch.cuda.is_available() else "cpu"
```

Reply:
Fixed